packages/actor-query-operation-orderby-sparqlee/test/ActorQueryOperationOrderBySparqlee-test.ts

```diff
@@ -146,3 +146,117 @@ describe('ActorQueryOperationOrderBySparqlee', () => {
+      }),
+    };
+  });
+
+
+  describe('An ActorQueryOperationOrderBySparqlee instance multiple comparators', () => {
+    let actor: ActorQueryOperationOrderBySparqlee;
+    let orderA: Algebra.TermExpression;
+    let orderB: Algebra.TermExpression;
+    let descOrderA: Algebra.OperatorExpression;
+    let descOrderB: Algebra.OperatorExpression;
+    let orderA1: Algebra.OperatorExpression;
+    let orderB1: Algebra.OperatorExpression;
+
+    beforeEach(() => {
+      actor = new ActorQueryOperationOrderBySparqlee({ name: 'actor', bus, mediatorQueryOperation });
+      orderA = { type: 'expression', expressionType: 'term', term: variable('a') };
+      orderB = { type: 'expression', expressionType: 'term', term: variable('b') };
+      descOrderA = { type: 'expression', expressionType: 'operator', operator: 'desc', args: [orderA] };
+      descOrderB = { type: 'expression', expressionType: 'operator', operator: 'desc', args: [orderB] };
+      orderA1 = { args: [orderA], expressionType: 'operator', operator: 'strlen', type: 'expression' };
+      orderB1 = { args: [orderB], expressionType: 'operator', operator: 'strlen', type: 'expression' };
+    });
+
+    // Testing implicit ascending with multiple comparators
+    it('should order A', async () => {
+      const op = { operation: { type: 'orderby', input: {}, expressions: [orderA] } };
+      const output = await actor.run(op);
+      const array = await arrayifyStream(ActorQueryOperation.getSafeBindings(output).bindingsStream);
+      expect(array).toMatchObject([
+        Bindings({ '?a': literal("Bosmans"),'?b': literal('Jos') }),
+        Bindings({ '?a': literal('Vermeulen'),'?b': literal('Jos') }),
+        Bindings({ '?a': literal('Vermeulen'),'?b': literal('Ben') }),
+      ]);
+    });
+
+    it('should order B', async () => {
+      const op = { operation: { type: 'orderby', input: {}, expressions: [orderB] } };
+      const output = await actor.run(op);
+      const array = await arrayifyStream(ActorQueryOperation.getSafeBindings(output).bindingsStream);
+      expect(array).toMatchObject([
+        Bindings({ '?a': literal('Vermeulen'),'?b': literal('Ben') }),
+        Bindings({ '?a': literal('Vermeulen'),'?b': literal('Jos') }),
+        Bindings({ '?a': literal("Bosmans"),'?b': literal('Jos') }),
+      ]);
+    });
+
+    it('should order priority B and secondary A, ascending', async () => {
+      const op = { operation: { type: 'orderby', input: {}, expressions: [orderB, orderA] } };
+      const output = await actor.run(op);
+      const array = await arrayifyStream(ActorQueryOperation.getSafeBindings(output).bindingsStream);
+      expect(array).toMatchObject([
+        Bindings({ '?a': literal('Vermeulen'),'?b': literal('Ben') }),
+        Bindings({ '?a': literal("Bosmans"),'?b': literal('Jos') }),
+        Bindings({ '?a': literal('Vermeulen'),'?b': literal('Jos') }),
+      ]);
+    });
+
+    // Testing descending with multiple comparators
```

Comment:
This comment _should_ not be needed. The essential information could be captured in the `it` label, so it's visible during test execution.